src/components/ui/icons/SpeakerLayout/index.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import Svg, { SvgProps } from '../Svg';
+
+const SpeakerLayout: React.SFC<SvgProps> = (props) => (
```

Comment:
React.SFC is deprecated now as per attached typescript guideline, should we start using `React.FunctionComponent` forward?
<img width="823" alt="Screen Shot 2021-08-12 at 5 53 18 PM" src="https://user-images.githubusercontent.com/61809205/129287991-0c93458f-6417-461e-8819-f41a0e8afd00.png">